test/lib/graph/helpers/ui5Framework.integration.js

```diff
@@ -32,20 +32,29 @@ test.beforeEach(async (t) => {
 		manifest: sinon.stub()
 	};
 
+	class Config {
+		static get typeDefs() {
+			return {};
+		}
+
+		async load() {}
+
+		get flat() {
+			return {};
+		}
+	}
+	sinon.stub(Config.prototype, "flat").returns({
```

Comment:
Why is it `returns` in here...

Reply:
Thanks! Fixed. Now everywhere is `value`.

Reply:
I can you explain why the tests still worked? Our code des not invoke "flat" as a function.

Reply:
It's defined within the original TS code as readonly property `flat` and it gets transpiled into ES6+ `get flat() {}`, which then can be accessed as a property and not a method.

That's why the stub should be `value`, but honestly I don't know why here it allowed a `returns`.

Reply:
I didn't really question why it can be stubbed that way but rather why no assertion failed. But I guess that's because in that test the values passed to pacote are not covered by asserts. As it's an integration test, I guess that's fine.